### DISCUSSION TITLE
Clarification strategy Minio / Aistor

### DISCUSSION THREAD

**Discussion by julienlau** [minio/minio#20787] [Q&A] [answered]
Hello,
Since your recent annoucement on AIstor I did not get what is the strategy about Minio.
Do you plan to abandon Minio and keep only an LTS ?
Do you plan to give Minio to some open source foundation and remove the AGPL lock ?
Do you plan an open source version of AIstor ?

Regards

**Comment by klauspost** [comment, marked as answer]
> Do you plan to abandon Minio and keep only an LTS ?

We plan to keep providing updates and fixes as we do now.

> Do you plan to give Minio to some open source foundation and remove the AGPL lock ?

No.

> Do you plan an open source version of AIstor ?

No.
